stable_baselines3/common/noise.py

```diff
@@ -81,7 +81,7 @@ def __call__(self) -> np.ndarray:
             + self._sigma * np.sqrt(self._dt) * np.random.normal(size=self._mu.shape)
         )
         self.noise_prev = noise
-        return noise
+        return noise.astype(np.float32)
```

Comment:
I would add the dtype in the constructor, defaulting to `np.float32` (so it can also work with other dtypes)